org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/manifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <!-- associated voice handler -->
+  <voice id="org.opent2t.sample.lightcontrol.cortana.vcd.superpopular" />
+  <!-- associated schemas -->
```

Comment:
You meant to add a description for the PR? added. :)

---
In reply to: [98082773](https://github.com/openT2T/translators/pull/149#discussion_r98082773) [](ancestors = 98082773)